### ISSUE TITLE
How do I avoid import name conflicts inside a dependency container?

### ISSUE THREAD

**Issue by Holofox** [Milad-Akarie/injectable#100]
Hello! I have two packages that have two classes with the same name. Inside the generated container with dependencies, they start to conflict, the problem was discovered after updating to the latest version of the package.

```console
lib/injection.config.dart:13:1: Error: 'Client' is imported from both 'package:centrifuge/src/client.dart' and 'package:grpc/src/client/client.dart'.
import 'package:grpc/grpc.dart';
^^^^^^
```
Any ideas?

**Comment by Milad-Akarie** [comment]
@Holofox This worked before?

**Comment by Milad-Akarie** [comment]
@Holofox fixed in injectable_generator: 1.0.3
